src/components/Questionnaire/QuestionnaireForm.tsx

```
@@ -860,7 +870,8 @@ export function QuestionnaireForm({
               clearError={(questionId: string) => {
                 setQuestionnaireForms((prev) =>
                   prev.map((f) =>
-                    f.questionnaire.id === form.questionnaire.id
+                    (f.questionnaire.instanceId || f.questionnaire.id) ===
+                    (f.questionnaire.instanceId || f.questionnaire.id)
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)